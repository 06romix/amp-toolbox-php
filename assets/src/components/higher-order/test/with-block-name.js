/**
 * External dependencies
 */
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import withBlockName from '../with-block-name';

describe( 'withBlockName', () => {
	it( 'provides the the wrapped component with block name as props', () => {
		const EnhancedComponent = withBlockName( () => (
			<div />
		) );

		const wrapper = shallow(
			<EnhancedComponent />
		);

		expect( wrapper.dive() ).toMatchSnapshot();
	} );
} );
