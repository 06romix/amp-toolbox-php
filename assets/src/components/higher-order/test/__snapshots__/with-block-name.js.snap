// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withBlockName provides the the wrapped component with block name as props 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
